docs/getting-started/clients/mobile/ios/index.mdx

```diff
@@ -300,3 +300,14 @@ Mac’s Sharing config.
 For example: if you want to debug the **iOS.Autofill** extension, you would complete steps 1 - 3,
 then go to your iOS device, open a browser, go to a login, tap the key icon and open Bitwarden from
 the bottom popup.
+
+### Using Server Tunneling
+
+Instead of having to deal with configuration on your device or emulator, you can create a proxy
+tunnel to you local server using `ngrok` and have your app connect to it directly.
+([More info in this server section](../../../server/tunnel.md#ngrok-requires-a-free-account))
```

Comment:
Either of the ingress tunnels should work.

```suggestion
Instead of configuration your device or emulator to ignore SSL certificates, you can instead use a [proxy tunnel to you local server](../../../server/tunnel.md) and have your app connect to it directly.
```

Reply:
I'll change on the android as well.